blog/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.contrib.auth import get_user_model
+
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
```

Comment:
why do u need this, will it has other names by default ?

Reply:
Deleted.